poi/src/main/java/org/apache/poi/hssf/record/TopMarginRecord.java

```diff
@@ -79,6 +79,6 @@ public HSSFRecordTypes getGenericRecordType() {
 
     @Override
     public Map<String, Supplier<?>> getGenericProperties() {
-        return GenericRecordUtil.getGenericProperties("magin", this::getMargin);
+        return GenericRecordUtil.getGenericProperties("margin", this::getMargin);
     }
 }
```

Comment:
I try not to fix missing newline at end of file warnings in PRs like this, although I prefer for them not to be missing.